frontend/test/__support__/testDataset.js

```diff
@@ -358,3 +358,43 @@ export default {
     },
   ],
 };
+
+export const testDatasetWithImage = {
```

Comment:
Can we use our mock factories `createMockDatasetData` and `createMockColumn` here to avoid type casting in the test?